accord-core/src/main/java/accord/utils/async/AsyncResults.java

```diff
@@ -201,6 +201,12 @@ public Throwable failure()
                 throw new IllegalStateException("Result succeeded");
             return result.failure;
         }
+
+        @Override
+        public String toString()
+        {
+            return getClass().getSimpleName() + "{status=" + (isDone() ? isSuccess() ? "success" : "failure" : "pending") + "}";
```

Comment:
im in the debugger all the time, so this is to make my life easier... I don't know if we log any results, but in a debugger there are many to look at so saves times